upgrading/automated_upgrades.adoc

```diff
@@ -546,12 +546,26 @@ OpenShift upgrades] for further guidance.
 [[special-considerations-for-mixed-environments]]
 ==== Special considerations for mixed environments
 
-Before you upgrade a mixed environment, for example one with Red Hat Enterprise
+Before you upgrade a mixed environment, such as one with Red Hat Enterprise
 Linux (RHEL) and RHEL Atomic Host, set values in the inventory file
 for both the `openshift_pkg_version` and `openshift_image_tag` parameters.
-Setting these variables ensures that all nodes in your cluster run the same
+Setting these values ensures that all nodes in your cluster run the same
 version of {product-title}.
 
+For example, to upgrade from {product-title}
+3.9 to {product-title} 3.10, you would set the values as follows:
+
+----
+openshift_pkg_version=v3.10.16
+openshift_image_tag=v3.10.16
+----
+
+[NOTE]
+====
+These options may also be present in other, non-mixed, environments.
```

Comment:
s/options may/parameters might

> ! may v
Use only to indicate permission. Otherwise, use “can” or “might.”
“May” is overused in technical information and should in most instances be changed to “can” or “might.”

Reply:
Done. Thanks.